### PR TITLE
docs: Changed message in the MakeFile

### PR DESCRIPTION
**Pull request by priyanka0906**
### Description

Link is outdated so it is modified with the new one.

### Related issue(s) and discussion
- Fixes #[ISSUE NUMBER]

### CODE REVIEW

**Comment by priyanka0906** [comment]
@alexgarel Please have a look.

File: Makefile [hunk 1]
```diff
@@ -40,7 +40,7 @@ goodbye:
 # Local #
 #-------#
 dev: hello build init_backend _up import_sample_data create_mongodb_indexes refresh_product_tags
-	@echo "🥫 You should be able to access your local install of Open Food Facts at http://productopener.localhost"
+	@echo "🥫 You should be able to access your local install of Open Food Facts at http://world.openfoodfacts.localhost//"
```
Comment (stephanegigandet) [inline Makefile:43]:
```suggestion
	@echo "🥫 You should be able to access your local install of Open Food Facts at http://world.openfoodfacts.localhost/"
```

Only 1 trailing / is needed: http://world.openfoodfacts.localhost/

Comment (priyanka0906) [reply Makefile]: @stephanegigandet  done

**Review by alexgarel — APPROVED** [review]
Good :-)

**Pull request merged**